src/plugins/plot/configuration/PlotSeries.js

```diff
@@ -410,7 +414,7 @@ export default class PlotSeries extends Model {
         const currentYVal = this.getYVal(point);
         const lastYVal = this.getYVal(data[insertIndex - 1]);
 
-        if (this.isValueInvalid(currentYVal) && this.isValueInvalid(lastYVal)) {
+        if ((this.isValueInvalid(currentYVal) && this.isValueInvalid(lastYVal))) {
```

Comment:
if this passes linting im cool but it might not like the extra parens

Reply:
yeah that was leftover, removing